lib/dpl/provider/now.rb

```diff
@@ -0,0 +1,136 @@
+module DPL
+    class Provider
+      class Now < Provider
+        npm_g 'now'
+  
+        def directory
+          File.expand_path( (context.env['TRAVIS_BUILD_DIR'] || '.' ) + '/' + (options[:directory] || '') )
+        end
+  
+        def deploy_name
+          options[:deploy_name]
+        end
+  
+        def team
+          if options[:team]
+            log "> Adding custom team scope #{options[:team]}"
+            now_team_option = "--team #{options[:team]}"
+          else
+            now_team_option = ''
+            log '> No custom team scope provided.'
+          end
+
+          return now_team_option
```

Comment:
Explicit `return`s seem inconsistent with other parts of this class, so let's drop them.